src/styles/main.scss

```
@@ -10,5 +10,76 @@ body {
   box-sizing: border-box;
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+.catalog {
+  display: grid;
+  justify-content: center;
+
+  grid-template-columns: repeat(4, 200px);
```

Great opportunity for variable.
```suggestion
  grid-template-columns: repeat(4, $card-width);
```